.buildkite/scripts/integration-tests.ps1

```diff
@@ -49,3 +51,7 @@ try {
         Write-Output "Cannot generate HTML test report: $outputXML not found"
     }
 }
+
+if ($TestsExitCode -ne 0) {    
```

Comment:
It's probably worth adding checks like this to all of your external commands.  (Annoying i know).  Otherwise it's like you're doing a `set +x` before each command and ignoring the results.

You have commands on lines 12, 13, and 28 in here that may or may not fail without you properly knowing.